modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/artifact/DeployableArtifactDescriptorFactoryTestCase.java

```diff
@@ -389,6 +405,53 @@ private void assertClassLoaderModelWithPluginDependencyAndAdditionalDependencies
     assertThat(dependantPluginDescriptor.getClassLoaderModel().getDependencies(), hasSize(1));
   }
 
+  @Test
+  public void classLoaderModelWithPluginDependencyAndSharedLibrariesLightweight() throws Exception {
+    assertClassLoaderModelWithPluginDependencyAndSharedLibraries("/plugin-dependency-with-shared-libraries-lightweight");
+  }
+
+  @Test
+  @Issue("MULE-19282")
+  public void classLoaderModelWithPluginDependencyAndSharedLibrariesInProfileLightweight() throws Exception {
+    assertClassLoaderModelWithPluginDependencyAndSharedLibraries("/plugin-dependency-with-shared-libraries-in-profile-lightweight");
+  }
+
+  @Test
```

Comment:
Missing Issue annotation

Reply:
this tests does not use profiles, i just added them to cover the sharedLibraries scenario which didn't have tests here.